docs/cli.md

```diff
@@ -506,6 +506,38 @@ gcli tag update --tag tagA --rename newTag
 gcli tag update --tag tagA --comment "new comment"
 ```
 
+### Topic commands
+
+#### Display a topic's details
+
+```bash
+gcli topic details --metalake metalake_demo --name kafka.default --topic topic3
```

Comment:
1. please remove the "--metalake", be consistent with others;
2. shall the "--name" be the full entity name, like "kafka.default.topic3"?

Reply:
The owner code has not been merged yet and that will be fixed when that happens.

We could use the full entity name and drop the --topic flag, but it may confuse users as catalogs and topics are at the same level. I am okay with either way to do it.

Reply:
As mentioned above @shaofengshi I am fine with it being either way, which do you prefer?